examples/tutorials/forced_alignment_for_multilingual_data_tutorial.py

```diff
@@ -0,0 +1,444 @@
+        i1 = i2
+
+    # compue word alignments from token alignments
+    separator = " "
+    words = []
+    i1, i2, i3 = 0, 0, 0
+    while i3 < len(transcript):
+        if i3 == len(transcript) - 1 or transcript[i3] == separator:
+            if i1 != i2:
+                if i3 == len(transcript) - 1:
+                    i2 += 1
+                s = 0
+                segs = segments[i1 + s : i2 + s]
+                word = "".join([seg.label for seg in segs])
+                score = sum(seg.score * seg.length for seg in segs) / sum(seg.length for seg in segs)
+                words.append(Segment(word, segments[i1 + s].start, segments[i2 + s - 1].end, score))
+            i1 = i2
+        else:
+            i2 += 1
+        i3 += 1
+    return segments, words
+
+
+# utility function for plotting word alignments
+def plot_alignments(segments, word_segments, waveform, input_lengths, scale=10):
+    fig, ax2 = plt.subplots(figsize=(64, 12))
+    plt.rcParams.update({"font.size": 30})
+
+    # The original waveform
+    ratio = waveform.size(0) / input_lengths
+    ax2.plot(waveform)
+    ax2.set_ylim(-1.0 * scale, 1.0 * scale)
+    ax2.set_xlim(0, waveform.size(-1))
+
+    for word in word_segments:
+        x0 = ratio * word.start
+        x1 = ratio * word.end
+        ax2.axvspan(x0, x1, alpha=0.1, color="red")
+        ax2.annotate(f"{word.score:.2f}", (x0, 0.8 * scale))
+
+    for seg in segments:
+        if seg.label != "|":
+            ax2.annotate(seg.label, (seg.start * ratio, 0.9 * scale))
+
+    xticks = ax2.get_xticks()
+    plt.xticks(xticks, xticks / sample_rate, fontsize=50)
+    ax2.set_xlabel("time [second]", fontsize=40)
+    ax2.set_yticks([])
+
+
+######################################################################
+# Aligning non-English data
+# -----------------------------
+#
+# Here we show examples of computing forced alignments of utterances in
+# 5 languages using the multilingual Wav2vec2 model, with the alignments visualized.
+# Here we first load the model and dictionary.
+#
+
+from torchaudio.models import wav2vec2_model
+
+model = wav2vec2_model(
+    extractor_mode="layer_norm",
+    extractor_conv_layer_config=[
+        (512, 10, 5),
+        (512, 3, 2),
+        (512, 3, 2),
+        (512, 3, 2),
+        (512, 3, 2),
+        (512, 2, 2),
+        (512, 2, 2),
+    ],
+    extractor_conv_bias=True,
+    encoder_embed_dim=1024,
+    encoder_projection_dropout=0.0,
+    encoder_pos_conv_kernel=128,
+    encoder_pos_conv_groups=16,
+    encoder_num_layers=24,
+    encoder_num_heads=16,
+    encoder_attention_dropout=0.0,
+    encoder_ff_interm_features=4096,
+    encoder_ff_interm_dropout=0.1,
+    encoder_dropout=0.0,
+    encoder_layer_norm_first=True,
+    encoder_layer_drop=0.1,
+    aux_num_out=31,
+)
+
+torch.hub.download_url_to_file(
+    "https://dl.fbaipublicfiles.com/mms/torchaudio/ctc_alignment_mling_uroman/model.pt", "model.pt"
+)
+checkpoint = torch.load("model.pt", map_location="cpu")
```

Comment:
Can we use `load_state_dict_from_url`? so that the tutorial won't leave intermediate files here and there.

https://pytorch.org/docs/stable/hub.html#torch.hub.load_state_dict_from_url